### ISSUE TITLE
wechaty v0.8.54 does not install all required component 

### ISSUE THREAD

**Issue by galendai** [Chatie/wechaty#522]
### Steps to reproduce the behavior (and fixes, if any)
1. Start a new project
2. `$ yarn add wechaty`
3. `$ cd node_modules/wechaty`
4. `$ tree`
```
├── CHANGELOG.md
├── LICENSE
├── README.md
├── bin
│   ├── doctor.ts
│   ├── entrypoint.sh
│   ├── io-client.ts
│   ├── pre-push
│   ├── version.ts
│   ├── xvfb-chromium
│   └── xvfb-run
├── dist
│   ├── bin
│   │   ├── doctor.d.ts
│   │   ├── doctor.js
│   │   ├── doctor.js.map
│   │   ├── io-client.d.ts
│   │   ├── io-client.js
│   │   ├── io-client.js.map
│   │   ├── version.d.ts
│   │   ├── version.js
│   │   └── version.js.map
│   └── package.json
├── node_modules
└── package.json
```

it seems some components were properly installed. this will cause missing of 'wechaty' module. error message below:
```
module.js:471
    throw err;
    ^

Error: Cannot find module '/Users/galendai/dev/wechaty-playground/run'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```
----
### Remark
manually replace `./node_module/wechaty/dist` from the `dist` of this repo by `$ npm run dist` could resolve the problem.

**Comment by chenzhutian** [comment]
v0.8.55 also

**Comment by zixia** [comment]
Sorry about this, it's related to https://github.com/Chatie/wechaty/commit/5b05801c3c1333f0e6d160e33d99222003eb0522

However, the bug had already been there long times ago. The reason of this is because we had copied the `package.json` to the `dist/` folder.

When run `npm pack`, npm will check every directory for `package.json`, if there's a `files` entry, then npm will only pack the files specified by the `files` entry.

We have not experiencing this bug before, is we also specified some rules like `*.js` so the files will be include. After we get rid of the useless rules, then the bug appear.
